internal/provider/nordvpn/provider.go

```diff
@@ -6,24 +6,25 @@ import (
 
 	"github.com/qdm12/gluetun/internal/constants/providers"
 	"github.com/qdm12/gluetun/internal/provider/common"
-	"github.com/qdm12/gluetun/internal/provider/nordvpn/updater"
+	"github.com/qdm12/gluetun/internal/provider/nordvpn/api"
 	"github.com/qdm12/gluetun/internal/provider/utils"
 )
 
 type Provider struct {
 	storage    common.Storage
 	randSource rand.Source
 	utils.NoPortForwarder
-	common.Fetcher
+	api    *api.API
+	warner common.Warner
```

Comment:
I believe we can stuff this back as it was and use the same interface `common.Fetcher` as before.